robot/EDA/resources/SystemSettingsPageObject.py

```diff
@@ -32,6 +32,43 @@ def update_system_dropdown_value(self,**kwargs):
                                                 error=f"'{value}' as dropdown value in '{field}' field is not available ")
             self.selenium.click_element(locator)
 
+    def enter_account_name_format(self,**kwargs):
+        """ This method will enter the account name format after selecting other in the drop down
+            Pass the expected value to be set in the input field as arguments
+        """
+        for field,value in kwargs.items():
+            locator = eda_lex_locators["eda_settings_system"]["other_accname_format"].format(field)
+            self.selenium.wait_until_page_contains_element(locator,
+                                                error=f"'{field}' field is not available ")
+            self.selenium.clear_element_text(locator)
+            self.selenium.get_webelement(locator).send_keys(value)
+
+    def select_recipient_type_value(self,**kwargs):
+        """ This method will select the drop down field value passed in keyword arguments
+            Pass the expected value to be selected in the drop down field from the tests
+        """
+        for field,value in kwargs.items():
+            locator = eda_lex_locators["eda_settings_system"]["recipient_type_value"].format(field,value)
+            self.selenium.wait_until_page_contains_element(locator,
+                                                error=f"'{value}' as dropdown value in '{field}' field is not available ")
+            self.selenium.click_element(locator)
+
+    def select_recipient(self,**kwargs):
+        """ This method will enter the account name format after selecting other in the drop down
+            Pass the expected value to be set in the input field as arguments
```

Comment:
Is this documentation correct ?

Reply:
Updated documentation to reflect the functionality